Add script for publishing singularity containers (#553)
AdaptiveCpp/AdaptiveCpp: install/scripts/README.md

```
Note that there are two type of installation scripts available at the moment the regular ones loacted in the `install/scripts/` directory, and scripts that use spack to install the dependencies located in `install/scripts/spack-install/`. The spack install scripts are well tested, therfore we recommend using those for the installation. The regular install scripts might need some changes to work flawlessly.

## Pre-built singularity containers

> We proved pre-built singularity images for all supported distributions (centos-7, ubuntu-18.04, ubuntu-20.04, archlinux). The containers are available through the following link: http://repo.urz.uni-heidelberg.de/sycl/singularity/ 

Reviewer: ```suggestion
We provide pre-built singularity images for all supported distributions. The containers are available here: http://repo.urz.uni-heidelberg.de/sycl/singularity/ 
```

> We test the provided images by building the hipSYCL unit tests for all supported backend, and in the case of Cuda and omp executing them.

Reviewer: ```suggestion
The images are validated by building the hipSYCL unit tests for all supported backends, and running them for OpenMP and CUDA.
```

> Please note that due to legal reasons, the images do not contain the Cuda installation. Please use the `install/scripts/install-Cuda.sh` script to install it afterwards. Note that this is only possible in case the container is writable; therefore we recommend installing Cuda by executing the following commands:

Reviewer: ```suggestion
Please note that due to legal reasons, the images do not contain the CUDA installation. Please use the `install/scripts/install-cuda.sh` script to install it afterwards. Note that this is only possible in case the container is writable; therefore we recommend installing CUDA by executing the following commands:
```
